design-toolkit/components/src/components/chip/styles.ts

```diff
@@ -64,11 +64,20 @@ export const DeletableChipStyles = tv({
   extend: BaseChipStyles,
   slots: {
     chip: [
-      'gap-xs outline-interactive',
-      'enabled:focus-visible:outline-interactive-hover',
-      'enabled:hover:outline-interactive-hover',
-      'disabled:fg-disabled disabled:cursor-not-allowed disabled:outline-interactive-disabled',
+      'gap-xs outline-interactive cursor-default',
+      'enabled:fg-primary-muted',
+      'disabled:fg-disabled disabled:cursor-not-allowed disabled:outline-interactive-disabled disabled:bg-interactive-disabled',
+      'has-[.remove:focus-visible]:fg-hover',
+      'has-[.remove:hover]:fg-hover',
+      'has-[.remove[data-pressed]]:fg-pressed',
+    ],
+    remove: [
+      'remove',
```

Comment:
I think we may want to consider replacing the plain RAC Button here with the DTK Button so styles don't need to be reimplemented. Once that happens, the `has-` selectors above can target the `.group/button` className instead of having to add an arbitrary `remove` class as a selector target